view.go

```diff
@@ -84,6 +97,13 @@ func updateTermRendered(style lipgloss.Style) (*glamour.TermRenderer, error) {
 }
 
 func PrintResultsWithStyle(results References, rootStyle lipgloss.Style) {
+	defer func() {
+		if rec := recover(); rec != nil {
+			fmt.Fprintf(os.Stderr, "An error occurred during PrintResultsWithStyle: %v\n", rec)
+			os.Exit(1)
+		}
+	}()
```

Comment:
Can the different recovery blocks be combined? I'm not sure we should be using this for regular error handling.